src/khoj/routers/api_chat.py

```diff
@@ -583,6 +586,45 @@ async def send_rate_limit_message(message: str):
             await conversation_command_rate_limiter.update_and_check_if_valid(websocket, cmd)
             q = q.replace(f"/{cmd.value}", "").strip()
 
+        if ConversationCommand.Summarize in conversation_commands:
```

Comment:
Can you add some unit tests for this in `test_openai_chat_actors.py` and ideally `test_offline_chat_actors.py`? If you can't get offline chat working, I can run these locally.